packages/components/src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -29,6 +29,11 @@ const DList = styled.dl<DListProps>`
         `}
       }
     `}
+  display: flex;
+  flex-direction: ${({ direction = 'column' }) => direction};
```

Comment:
Kanskje vi kan gjøre noe ala:
```suggestion
  flex-direction: column;
  &:not(:has(> dt):has(> dd)) {
    flex-direction: ${({ direction = 'column' }) => direction};
  }
```
Litt stygt da 🙈 Men dette gjør i hvert fall at det kun er mulig å bruke `direction` når vi ikke har `dt` og `dd` som direkte barn. Altså at de må ligge i en `DescriptionListGroup` eller lignende.

Hvis vi går for dette burde det nok sikkert stå i description på `direction` at den kun fungerer om alt ligger i `DescriptionListGroup`s.

Reply:
Fixed nå. Også oppdatert dokumentasjonen.